src/components/SchedulePage/Schedule.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TimeSlot from '../SchedulePage/Timeslot';
+
+const hothDate = '2021-02-23'; // replace with real date later
+const utcOffset = '-08:00'; // PST
+const events = [
+	{
+		startTime: new Date(hothDate + 'T09:30:00' + utcOffset),
```

Comment:
Sorry, I know this has already been discussed, but I think it would generally be cleaner to use a function rather than duplicating the concatenation logic. Consider:

```suggestion
// time is a string of the form "HH:mm:ss"
function getDateObj(time) {
  return new Date(`${hothDate}T${time}${utcOffset}`);
}

const events = [
	{
		startTime: getDateObj('09:30:00'),
```

Additionally, you might even want to look into sharing the `hothDate` variable somehow with https://github.com/uclaacm/hoth.uclaacm.com/blob/581b54a8d3794ed0578370af2977a79a86c3e2b3/src/components/HomePage/Banner.js#L14

Reply:
I like both of these suggestions. Thanks @TimothyGu

Reply:
Also, @TimothyGu should we just add another file for HoTH "metadata" or something? (probably in another PR?)